test/e2e/ha/core_contrail_services_test.go

```diff
@@ -378,6 +378,11 @@ func assertReplicasReady(t *testing.T, w wait.Wait, r int32) {
 		assert.NoError(t, w.ForReadyStatefulSet("hatest-webui-webui-statefulset", r))
 	})
 
+	t.Run(fmt.Sprintf("then a ContrailCNI Job has %d completions", r), func(t *testing.T) {
```

Comment:
I'm not sure if this check should be in assertReplicasReady. Let assume that one node will fail. I think completed job cnt will be still 3, however here we are checking against 2 in failover scenario

Reply:
I think that this test will never pass